lib/ejs.js

```diff
@@ -191,7 +196,11 @@ function includeSource(path, options) {
   if (!opts.filename) {
     throw new Error('`include` requires the \'filename\' option.');
   }
-  includePath = exports.resolveInclude(path, opts.filename);
+  if(path.substr(0,1) === "/" && typeof opts.root == "string" && opts.root != ""){
```

Comment:
Also, if you're doing the same conditional check twice, maybe make it a reusable private utility function.